src/main/java/frc/robot/Constants.java

```diff
@@ -63,15 +63,15 @@ public static final class Swerve {
 
         /* Swerve Profiling Values */
         public static final double maxSpeed = 4.5; //meters per second (default = 4.5)
-        public static final double maxAngularVelocity = 11.5;
+        public static final double maxAngularVelocity = 8;   //11.5
```

Comment:
We might consider modifying this value in TeleopSwerve instead of here. It just depends whether or not we think it would be useful to use the original velocity for auto-related movement and rotation in RotateToHeading. For now you might just add a comment here to mention that this was lowered at the driver's request for ease of use during teleop.